aries-site/src/examples/templates/dashboards/components/Measure.js

```diff
@@ -0,0 +1,34 @@
+import { cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { Box, NameValuePair, Text } from 'grommet';
+
+export const Measure = ({ name, value, onClick }) => {
+  const { icon, label } = name;
+  const adjIcon = cloneElement(icon, { size: 'small' });
```

Comment:
What does `adjIcon` mean?

Reply:
adjustedIcon

Reply:
would something like this read better?
```
  const { icon: iconProp, label } = name;
  const icon = cloneElement(iconProp, { size: 'small' });
```

Reply:
Changed to the above

Reply:
I like what you changed it to makes more sense